test/integration/pkg/generated/openapi_generated.go

```diff
@@ -1015,9 +1015,7 @@ func schema_test_integration_testdata_valuevalidation_Foo(ref common.ReferenceCa
 	return common.OpenAPIDefinition{
 		Schema: spec.Schema{
 			SchemaProps: spec.SchemaProps{
-				Type:          []string{"object"},
-				MinProperties: ptr.To[int64](1),
-				MaxProperties: ptr.To[int64](5),
+				Type: []string{"object"},
```

Comment:
I must have missed something.  Why is this change needed?

Reply:
ValidateType was adjusted to continue validation for non-opaque struct types (mostly to enable recursive validation of the new subschema fields):

https://github.com/kubernetes/kube-openapi/pull/469/files/626a771d522a239689b64c1b50ee0ebf37215d8b#diff-72f8260914152fe9557b05d756ec94854184645e991ff755855628428d6d7a18L180-R294

The existing test for this had used `minProperties` and `maxPropertes` on a struct (rather than map), which now results in error.

I somewhat arbitrarily chose to continue this restricting (rather than widening those fields to be used for structs) because it seemed to me there did not seem to be much of a use case. Happy to allow them for structs again if someone else things it could be useful

Reply:
Ended up allowing MinProperties/MaxProperties since it seemed unnecessary restriction and to keep this PR focused